src/Points/points_fastapi.py

```
@@ -95,6 +96,18 @@ def save_point(
 
         return {"success": True, "message": 'Successfully saved the point'}
     
+    except IntegrityError:
+        raise HTTPException(
+            status_code=409,
+            detail={
+                "success": False,
+                "message": "A point already exists with that name, use a different name."
+            }
+        )
```

Ruff (0.16.0)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)